src/main/java/com/stuypulse/robot/subsystems/Edwin.java

```diff
@@ -0,0 +1,229 @@
+package com.stuypulse.robot.subsystems;
+
+import com.kauailabs.navx.frc.AHRS;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.stuypulse.stuylib.control.Controller;
+import com.stuypulse.stuylib.control.feedback.PIDController;
+import com.stuypulse.stuylib.control.feedforward.Feedforward;
+import com.stuypulse.stuylib.math.Angle;
+
+import static com.stuypulse.robot.Constants.*;
+import static com.stuypulse.robot.Constants.EdwinSettings.*;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.DifferentialDriveKinematics;
+import edu.wpi.first.math.kinematics.DifferentialDriveOdometry;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj.smartdashboard.Field2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Edwin extends Robot {
+    
+    private final MotorControllerGroup left;
+    private final MotorControllerGroup right;
+    
+    private final DifferentialDrive drivetrain;
+
+    private final Encoder leftGreyhill;
+    private final Encoder rightGreyhill;
```

Comment:
replace these with relative encoders and useeeeeeeeeeeeeeeee one of them. idk if it matters which one u pick as long as it's the same one (front, middle, or back) on both sides.  see which one the og edwin repo picked, i'm guessing it was middle